### PR TITLE
Add more links and corrections to user account management

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

- Add links where appropriate 
- Minor corrections

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/18283
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: privilege-management.md [hunk 1]
```diff
@@ -133,7 +169,6 @@ REVOKE ALL PRIVILEGES ON `test`.* FROM 'genius'@'localhost';
 >
 > `REVOKE` 收回权限时只做精确匹配，若找不到记录则报错。而 `GRANT` 授予权限时可以使用模糊匹配。
```
Comment (Oreoxmt) [inline privilege-management.md:171]:
```suggestion
```

File: privilege-management.md [hunk 2]
```diff
@@ -561,10 +574,4 @@ User+Host 可能会匹配 `user` 表里面多行，为了处理这种情况，`u
 
 TiDB 启动时，将一些权限检查的表加载到内存，之后使用缓存的数据来验证权限。系统会周期性的将授权表从数据库同步到缓存，生效则是由同步的周期决定，目前这个值设定的是 5 分钟。
```
Comment (qiancai) [inline privilege-management.md:575]:
```suggestion
TiDB 启动时，会将一些权限检查的表加载到内存，之后使用缓存的数据来验证权限。执行权限管理语句（如 `GRANT`、`REVOKE`、`CREATE USER` 和 `DROP USER`）将立即生效。
```

File: privilege-management.md [hunk 3]
```diff
@@ -561,10 +574,4 @@ User+Host 可能会匹配 `user` 表里面多行，为了处理这种情况，`u
...
 
-修改了授权表，如果需要立即生效，可以手动调用：
-
-{{< copyable "sql" >}}
-
-```sql
-FLUSH PRIVILEGES;
-```
+修改了授权表，如果需要立即生效，可以手动执行 [`FLUSH PRIVILEGES`](/sql-statements/sql-statement-flush-privileges.md)。
```
Comment (qiancai) [inline privilege-management.md:577]:
```suggestion
使用 `INSERT`、`DELETE`、`UPDATE` 等语句手动修改 `mysql.user` 等授权表不会立即生效。该行为与 MySQL 兼容。如需立即生效，可以手动执行 [`FLUSH PRIVILEGES`](/sql-statements/sql-statement-flush-privileges.md) 语句更新权限的缓存。
```

File: privilege-management.md (hunk 2 above)
Comment (qiancai) [reply privilege-management.md:575]: 看起来 [pingcap/docs@`fce19a3` (#1089)](https://github.com/pingcap/docs/pull/1089/commits/fce19a352d1dff3004bb450801e3ac9008363cd7) 提交的时间更晚些

File: user-account-management.md [hunk 4]
```diff
@@ -14,42 +14,34 @@ summary: TiDB 用户账户管理主要包括用户名和密码设置、添加用
...
 ## 添加用户
 
 添加用户有两种方式：
 
-* 通过标准的用户管理的 SQL 语句创建用户以及授予权限，比如 `CREATE USER` 和 `GRANT`。
-* 直接通过 `INSERT`、`UPDATE` 和 `DELETE` 操作授权表。不推荐使用这种方式添加用户，因为容易导致修改不完整。
-
-除以上两种方法外，你还可以使用第三方图形化界面工具来添加用户。
+* 通过标准的用户管理的 SQL 语句创建用户以及授予权限，比如 [`CREATE USER`](/sql-statements/sql-statement-create-user.md) 和 [`GRANT`](/sql-statements/sql-statement-grant-privileges.md)。
+* 直接通过 [`INSERT`](/sql-statements/sql-statement-insert.md)、[`UPDATE`](/sql-statements/sql-statement-update.md) 和 [`DELETE`](/sql-statements/sql-statement-delete.md) 操作授权表。不推荐使用这种方式添加或修改用户，因为容易导致修改不完整。
```
Comment (qiancai) [inline user-account-management.md:36]:
```suggestion
* 直接通过[`INSERT`](/sql-statements/sql-statement-insert.md)、[`UPDATE`](/sql-statements/sql-statement-update.md) 和 [`DELETE`](/sql-statements/sql-statement-delete.md) 操作授权表，然后执行 [`FLUSH PRIVILEGES`](/sql-statements/sql-statement-flush-privileges.md)。不推荐使用这种方式添加或修改用户，因为容易导致修改不完整。
```

File: user-account-management.md [hunk 5]
```diff
@@ -14,42 +14,34 @@ summary: TiDB 用户账户管理主要包括用户名和密码设置、添加用
...
+除以上两种方法外，你还可以使用[第三方图形化界面工具](/develop/dev-guide-third-party-support.md#gui)来添加用户。
 
 ```sql
 CREATE USER [IF NOT EXISTS] user [IDENTIFIED BY 'auth_string'];
 ```
 
-设置登录密码后，`auth_string` 会被 TiDB 经过加密存储在 `mysql.user` 表中。
-
-{{< copyable "sql" >}}
+设置登录密码后，`auth_string` 会被 TiDB 经过加密存储在 [`mysql.user`](/mysql-schema.md) 表中。
```
Comment (qiancai) [inline user-account-management.md:44]:
```suggestion
设置登录密码后，`auth_string` 会被 TiDB 加密并存储在 [`mysql.user`](/mysql-schema.md) 表中。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**